lib/mapfiles/map-file.ts

```diff
@@ -60,13 +103,7 @@ export class MapFileReader {
         }
     }
 
-    /**
-     *
-     * @param {(string|boolean)} segment
-     * @param {number} address
-     * @returns {(object | boolean)}
-     */
-    getLineInfoByAddress(segment, address) {
+    getLineInfoByAddress(segment: string | boolean, address: number) {
         for (let idx = 0; idx < this.lineNumbers.length; idx++) {
```

Comment:
Lots of index-based loops in this code but again trying to keep changes minimal

Reply:
It was written a long time ago in 2017, I just did what I knew would work :D

Reply:
Perfect for 2017! :P